lib/runtime/wasmer/imports_test.go

```diff
@@ -232,6 +232,59 @@ func Test_ext_offchain_local_storage_clear_version_1_Local(t *testing.T) {
 	require.Nil(t, val)
 }
 
+func Test_ext_offchain_http_request_start_version_1(t *testing.T) {
+	inst := NewTestInstance(t, runtime.HOST_API_TEST_RUNTIME)
+
+	encMethod, err := scale.Marshal([]byte("GET"))
+	require.NoError(t, err)
+
+	encUri, err := scale.Marshal([]byte("https://chainsafe.io"))
```

Comment:
Can you start an HTTP mock server and request that? Because this test will fail without the internet or if Chainsafe domain is down.

Reply:
the host function does not execute any request so this is just a URI formatted string to the runtime